packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -633,7 +633,13 @@ class _CupertinoTextFieldState extends State<CupertinoTextField> with AutomaticK
           rowChildren.add(
             GestureDetector(
               onTap: widget.enabled ?? true
-                  ? () => _effectiveController.clear()
+                  ? () {
+                // special handle onChanged for ClearButton.
```

Comment:
Indent everything forward so it's lined up with the ? above + 2. 

`// Also call onChanged when the clear button is tapped.`

Reply:
ok